### PR TITLE
Don't make the logo so dominant

### PR DESCRIPTION
**Pull request by ocaisa**
... and this way it also scale according to the device

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-![EESSI logo](img/logos/EESSI_logo_horizontal.png)
+<p align="center">
+  <img width="60%" src="img/logos/EESSI_logo_horizontal.png" alt="EESSI logo">
```
Comment (boegel) [inline docs/index.md:2]: Did you test this, does it work in a deployed version of the docs?

Comment (boegel) [reply docs/index.md:2]: See also https://github.com/mkdocs/mkdocs/issues/991 (not sure if that's fixed)

Comment (ocaisa) [reply docs/index.md:2]: Nope, didn't test, assumed the CI would that for me.

If there's a related bug, I can just use an absolute link

Comment (ocaisa) [reply docs/index.md:2]: It rendered fine in the editor

Comment (ocaisa) [reply docs/index.md:2]: According to [this comment](https://github.com/mkdocs/mkdocs/issues/991#issuecomment-233213340) it should work fine

Comment (ocaisa) [reply docs/index.md:2]: Tested, works

**Pull request merged**